src/common/http.ts

```diff
@@ -37,9 +37,21 @@ export function getStoreUrl(): string {
   return envUrl ? new URL(envUrl).href : new URL('https://superface.ai/').href;
 }
 
-export async function fetch(url: string, type: ContentType): Promise<Response> {
+export async function fetch(
+  url: string,
+  type: ContentType,
+  query?: Record<string, string | number>
+): Promise<Response> {
   const userAgent = `superface cli/${VERSION} (${process.platform}-${process.arch}) ${process.release.name}-${process.version} (with @superfaceai/one-sdk@${SDK_VERSION}, @superfaceai/parser@${PARSER_VERSION})`;
   try {
+    if (query) {
+      return superagent
+        .get(url)
+        .query(query)
+        .set('Accept', type)
+        .set('User-Agent', userAgent);
+    }
```

Comment:
I don't quite like this. Wouldn't it be better to build the thing like `let client = superagent.stuff()` and then optionally add the query into it if it's defined like `if (query) { client = client.query(query) }`?

Reply:
Yep, 👍 i will look into it. I wasn't able to get this working on the first try and I know there were some bugs in superagent.

Reply:
SuperAgent sucks.` let client = superagent.stuff()` will fire the request so I can't use that.